src/xdocs/checks/design/innertypelast.xml

```diff
@@ -29,22 +29,23 @@
         </source>
         <p id="Example1-code">Example:</p>
         <source>
-class Test {
-    private String s; // OK
-    class InnerTest1 {}
-    public void test() {} // violation, method should be declared before inner types.
+// violation message - method should be declared before inner types.
```

Comment:
please remove this line.

you have vilation comment on lines where checkstyle place them, it is enough.

Reply:
Sorry about that....
Done